Makefile

```diff
@@ -233,6 +233,12 @@ check:
 	@echo "goimports"
 	@! goimports -l . | grep -vF 'No Exceptions'
 
+.PHONY: unused
+unused:
+# The -fields option generates false-positives for embedded mutex and
+# related types. See https://github.com/dominikh/go-unused/issues/18.
```

Comment:
I'll file an issue for that and see if it's possible to extract the "export" comment, thanks.